code/modules/projectiles/guns/ds13/contactbeam.dm

```diff
@@ -312,24 +312,3 @@
 			new /obj/effect/effect/expanding_circle(location, 0.25, 1. SECONDS)
 			sleep(rand_between(2,4))
 		qdel(src)
-
-
-
-/*
-	Acquisition
-*/
-/decl/hierarchy/supply_pack/mining/contact_energy
```

Comment:
see previous comment regarding the placement of these decls.

I've been asked about it and defended their current position, opposed to moving. This does not help work efficiency

Reply:
We have also made it clear before that this is not how the system works and that you should stop doing it. Yet you persist.